extensions/circuit/src/adapters/alu.rs

```diff
@@ -119,8 +119,7 @@ pub struct WomBaseAluAdapterCols<T, const READ_32BIT_WORDS: usize, const WRITE_B
     pub rs2: T,
     /// 1 if rs2 was a read, 0 if an immediate
     pub rs2_as: T,
-    pub reads_aux: [[T; 2]; READ_32BIT_WORDS],
-    pub writes_aux: [T; WRITE_BYTES],
+    pub write_mult: T,
```

Comment:
So my understanding is that each ALU instruction exists in (at least) 2 variants: On 32 or 64 Bit words. In the 64-Bit case, we need to do 2 memory accesses to the 32-Bit memory. But that applies to the write as well then, so the number of write multiplicities would have to be generic as well then?